src/l1/mod.rs

```diff
@@ -528,6 +528,8 @@ fn start_watcher(
                     err
                 );
             }
+            // TODO make configurable
+            sleep(Duration::from_millis(1750)).await;
```

Comment:
a comment explaining where the value come from could help to understand more of the context :)